lib/runner/extensions/request.command.js

```diff
@@ -13,31 +13,41 @@ var _ = require('lodash'),
      * @param callback
      */
     resolveFiles = function (request, resolver, triggers, callback) {
-        var formParams = _.get(request, 'body.formdata');
+        var formParams = _.get(request, 'body.formdata'),
+            fileData = _.get(request, 'body.file');
 
-        if (!formParams) { return callback(null, request); }
+        if (!(formParams || fileData)) { return callback(null, request); }
```

Comment:
do stuff based on mode, else two callbacks would be called + if binary data is there, but form data is used in mode, both will get uploaded.